main.cpp

```diff
@@ -50,86 +50,105 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
+
+
+/* My exampes */
+
+/*apologies some were a bit tongue-in-cheek as I was running out of things to think of in the context of a working day - I don't really have a 5PM alarm! */
+
+
+//  1)
+//  Noun:       spindle
+//  action 1:   spindle moves in X
+spindle.moveX();
+//  action 2:   spindle checks on/off status
+spindle.isSpinning(); 
+//  action 3:   spindle maintains set speed
+spindle.maintainSpeed();
+
+//  2)
+//  Noun:       motorZ1
+//  action 1:   check position of encoder
+motorZ1.checkAngle();
+//  action 2:   calculate speed of motor z1
+motorZ1.CalculateSpeed(); 
+//  action 3:   motorZ1 slows to a stop
+motorZ1.stop();
+
+//  3)
+//  Noun:     vacuum  
+//  action 1:   check if vacuum is on
+vacuum.isOn();
+//  action 2:   vacuum is full
+vacuum.isFull(); 
+//  action 3:   vacuum issues warning
+vacuum.warning();
+
+//  4)
+//  Noun:       tool
+//  action 1:   get current cutting tool number
+tool.getCurrentNumber();
+//  action 2:   change curent tool
+tool.change(); 
+//  action 3:   check tool height
+tool.zeroHeight();
+
+//  5)
+//  Noun:       workshopDoor
+//  action 1:   workshop door is closed?
+workshopDoor.isClosed();
+//  action 2:   workshop door is moving
+workshopDoor.isMoving(); 
```

Comment:
`is` is a bad action word to use
From the instructions:
```
 2) for each of the 10 nouns, write out 3 actions it might perform, in plain english.
    a) AVOID state-checking actions, like "Car has a horn".  i.e.
        car.hasHorn()
    b) We use 'can' and 'has' to check the state of objects, so they aren't really 'action' verbs.
        "the Car blows its horn" is a better action, because the car is doing something.  its state is not being checked.  i.e.
            car.blowsHorn()
```
- [ ] revise